src/client/components/pages/entity-revisions.js

```diff
@@ -113,15 +87,23 @@ class EntityRevisions extends React.Component {
 	 * @returns {ReactElement} a HTML document which displays the Revision page
 	 */
 	render() {
-		const {revisions} = this.props;
-
 		return (
-			<div>
+			<div className="pageWithPagination">
```

Comment:
This should be `id` rather than `className`, I just realized

Reply:
I'll fix it everywhere